inc/task.class.php

```diff
@@ -334,19 +334,22 @@ function getTaskjobstatesForAgent($agent_id, $methods = array(), $options=array(
          //Cancel the job it has already been sent to the agent but the agent did not replied
          if ($result['run']['state'] == $jobstate::SERVER_HAS_SENT_DATA
                  or $result['run']['state'] == $jobstate::AGENT_HAS_SENT_DATA) {
-            $jobstates_to_cancel[$jobstate->fields['id']] = array(
+            $jobstates_to_cancel[$jobstate->fields['id']] = [
                'jobstate' => $jobstate,
-               'reason'   => __("The agent is requesting a configuration that has already been sent to him by the server. It is more likely that the agent is subject to a critical error.", 'fusioninventory'),
-               'code' => $jobstate::IN_ERROR
-            );
+               'reason'   => __("The agent is requesting a configuration that has already been sent to him by the server. It is more likely that the agent is subject to a critical error.",
```

Comment:
Are you sure the tool extract gettext works with return lines ?

Reply:
yes